kernel/kata_config_version

```diff
@@ -1 +1 @@
-49
+50
```

Comment:
I see you have bumped the version here but still CI is failing with: 
```
23:01:19 Kernel directory has changes check kernel/kata_config_version changed
23:01:19 ERROR: Please bump version in kernel/kata_config_version
23:01:19 Makefile:24: recipe for target 'test' failed
```

Reply:
@jcvenegas any idea of this failure?

Reply:
@jcvenegas and @nitkon, I just fixed it